test/SetGame.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  SetGame.swift
+//  test
+//
+//  Created by apple on 05/06/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import Foundation
+
+class SetGame
+{
+    var inPlayCards = [Card]()
+    
+    var deck = [Card]()
+    
+    var score = 0
+    
+    var selectedCards = [Int:Card]()
+    
+    var isSetCards = [Int:Card]()
+    
+    func cardGenerator()
+    {
+        var identifier = 0
+        for color in Card.CardColor.allValues
+        {
+            for simbol in Card.CardSimbol.allValues
+            {
+                for number in Card.CardNumber.allValues
+                {
+                    for shading in Card.CardShading.allValues
+                    {
```

Comment:
IF you have time you could replace the for in for logic, and use something like the mapping with the map function